lusidtools/jupyter_tools/notebook_stopper.py

```diff
@@ -0,0 +1,12 @@
+class StopExecution(Exception):
```

Comment:
maybe rename the file to `stop_execution.py` so its easier to find/know what is in the file

Reply:
Done
See https://github.com/finbourne/lusid-python-tools/pull/131